### ISSUE TITLE
libDaisy submodule capitalization

### ISSUE THREAD

**Issue by CorvusPrudens** [electro-smith/DaisyExamples#220]
The libDaisy capitalization is inconsistent across the tools in DaisyExamples. The submodule is set to always clone into `libdaisy`, and some of the `helper.py` commands create lower case libDaisy references. Since the repo name is libDaisy, it's probably best to get everything aligned with that capitalization.

This was particularly annoying for mac / linux users because the unix cpp configuration for vscode assumed a capital D.

**Comment by stephenhensley** [comment]
this should be resolved, but I know people _may_ still run into an issue with previous clones (though at this point, even with wrong spelling it compiles on the mac OS machine I was testing on ... So we'll see I suppose).

I'll leave this open for a bit with this note:

**If you get a `can't find libDaisy` related error -- you can either manually rename the "libdaisy" folder to "libDaisy", or reclone the repo to resolve the issue.**

**Comment by MrBlueXav** [comment]
Hello, I'm on a linux platform.
It's a problem when using helper.py for creating a new project.
The makefile won't work as it refers to "libdaisy" instead of libDaisy.
Easy to fix, though.

**Comment by HeckHeckHeckHeck** [comment]
The macos filesystem is acutally case insensitive

**Comment by CorvusPrudens** [comment]
@HeckHeckHeckHeck It would be more accurate to say that APFS _can_ be case-sensitive, but it is not by default. [See here for details](https://support.apple.com/lt-lt/guide/disk-utility/dsku19ed921c/mac).

**Comment by HeckHeckHeckHeck** [comment]
True, you are right! thanks for pointing that out. i actually forgot that since, case-insensitive is the default.
